src/Agent/NewRelic/Agent/Core/Aggregators/ErrorTraceAggregator.cs

```diff
@@ -41,6 +42,19 @@ public override void Dispose()
             _readerWriterLock.Dispose();
         }
 
+        protected override TimeSpan HarvestCycle
+        {
+            get
+            {
+                if (_configuration.ErrorTracesHarvestCycle.HasValue)
+                {
+                    return _configuration.ErrorTracesHarvestCycle.Value;
+                }
+
+                return DefaultHarvestCycle;
```

Comment:
Should we move the default value into DefaultConfiguration so that the logic for the harvest cycle is in just one location?

Reply:
I considered that (and did that in my first attempt), but I didn't want to lock the different cycles to one setting (1 minute).  I thought it would be more flexible this way.  If we want to centralize things, I can totally do that.

Reply:
What I'm thinking of is that `ErrorTracesHarvestCycle` should always have a value. If an override is present, it will return the override value. If no override is present, then it will return what should be the default value. I'm not describing having a single setting that controls all of the harvests.